be/src/storage/rowset/rowset_writer.h

```diff
@@ -124,6 +124,7 @@ class RowsetWriter {
 
 protected:
     vectorized::DictColumnsValidMap _global_dict_columns_valid_info;
+    const uint64_t MAX_SEGMENT_SIZE = static_cast<uint64_t>(config::max_segment_file_size * config::segment_file_scale);
```

Comment:
I have question, why does it need two configuration to determinate segment size?
Why not use only one configuration?

Reply:
I also have no idea why there is a ratio, maybe for the inaccuracy of segment size calculation. I just keep the original logical.

Reply:
I also agree to just use only one configuration. I will change it.